l2/system-contracts/Utils.sol

```diff
@@ -0,0 +1,14 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
+pragma solidity >=0.8.0;
+
+/**
+ * @author Matter Labs
+ * @dev Common utilities used in zkSync system contracts
+ */
+library Utils {
+    function safeCastToU24(uint256 x) internal pure returns (uint24) {
+        require(x < 2**24, "Overflow");
+
+        return uint24(x);
+    }
+}
```

Comment:
I'd avoid creating files called "utils". Everyone does it but it isn't great. There's no shame in having a small `SafeCasting.sol` file, same for other future "utils".